app/src/main/res/layout/choose_master_pswrd_fragment.xml

```diff
@@ -72,6 +72,29 @@
 
         </com.google.android.material.textfield.TextInputLayout>
 
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/add_a_hint"
+            style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="16dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/confirm_pswrd">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/hint_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_hint"
+                android:inputType="textNoSuggestions"
```

Comment:
Remove this as well. 
Reason for no suggestion for pasword is so that user dont use commonly used words as a part of subtext. its very convenient to use dictionary words.
But here its logical to allow user to use suggestions.